mitdevices/reticon120.c

```diff
@@ -84,7 +84,7 @@ EXPORT int reticon120___init(struct descriptor *niddsc_ptr __attribute__ ((unuse
     int f_nid = p_nid + 1;
     int fs_nid = f_nid + 1;
     ReturnOnError(DevFloat(&p_nid, &period), RETICON$_BAD_PERIOD);
-    for (i = 0; periods[i] < period && i < 15; i++) ;
+    for (i = 0; i < 15 && periods[i] < period; i++) ;
```

Comment:
I would rather see (). I can never remember the operator precedence accept that people often get it wrong.  How a a about: 
 for (i = 0; (i < 15) && (periods[i] < period); i++) ;

Reply:
easy peasy